src/components/Nav/ApplicationNav.vue

```diff
@@ -108,6 +110,8 @@ export default {
 
     <HelpMenu />
 
+    <ChatLauncher />
```

Comment:
```suggestion
    <ChatLauncher v-if="isCloud" />
```

This should only be on the nav bar if we're in a Cloud context.